frontend/public/components/marketplace/catalog-tiles.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Breadcrumb, ModelessOverlay, Button, Modal } from 'patternfly-react';
+import { CatalogTile, CatalogTileBadge, CatalogTileView, CatalogTileViewCategory } from 'patternfly-react-extensions';
+
+import { mockTileItems } from './mockItems';
+import { MarketplaceToolbar } from './toolbar';
+import { MarketplaceModelessOverlay } from './modeless-manager';
+import { interpolateMagma } from 'd3';
```

Comment:
I don't think this is used on this page. We should run these changes through a linter / automated cleanup.